src/api/users/src/models/celebrateSchema.js

```diff
@@ -1,5 +1,13 @@
 const { celebrate, Joi, Segments } = require('celebrate');
 
+const validateGetParams = () =>
+  celebrate({
+    [Segments.PARAMS]: {
+      per_page: Joi.string(),
```

Comment:
These should both be numbers, not strings

Reply:
Weird. I just tested this and it's working fine as `number`. I'm not sure if you remember but I had to make:
```
const validateId = () =>
  celebrate({
    [Segments.PARAMS]: {
      id: Joi.string().required(), <---- this a string
    },
  });
```
`id` into a string. Must be because I'm parsing `page` and `perPage` as ints whereas I'm not parsing `id` into an int, causing Firebase throws when attempting to get a doc using a number instead of a string (e.g. `Error: Value for argument "documentPath" is not a valid resource path. Path must be a non-empty string.`)

Cool though, thanks for the suggestion.